roles/docker/tasks/main.yml

```diff
@@ -36,9 +36,8 @@
     state: installed
     allow_downgrade: true
   loop:
-    # https://groups.google.com/g/xnat_discussion/c/yyPBkN4kayE/m/LUe5GQH5AAAJ
-    - docker-ce-24.0.9
-    - docker-ce-cli-24.0.9
+    - docker-ce-{{ docker_version }}
```

Comment:
will `latest` will work here?

Reply:
I don't think so. Would want to make it conditional on the variable value, if it's `latest` just use the package name, if it's anything else add in `-{{ docker_version }}`